gravitee-apim-console-webui/src/management/management.module.ts

```diff
@@ -40,6 +41,7 @@ import { GioPermissionModule } from '../shared/components/gio-permission/gio-per
     ApiCreationModule,
     GioPolicyStudioRoutingModule.withRouting({ stateNamePrefix: 'management.apis.detail.design.flowsNg' }),
     EnvironmentApplicationModule,
+    ApisModule,
```

Comment:
Do you want to add naming convention at this lvl ? or no (we can see it later 🤷‍♂️ )
btw : ApiCreationModule, ApisModule, EnvironmentApplicationModule 
@gcusnieux

Reply:
@gcusnieux & @ThibaudAV I propose you to keep it in mind for the next pull requests. For sure we can discuss it and I will make the changes if necessary :)